src/libyang/patch/series

```diff
@@ -1,2 +1,3 @@
 libyang.patch
+libyang_mgmt_framework.patch
```

Comment:
In the Future, are we planning to put all patches related to a framework in this single file?,
 If not, I feel it is better to have a file name based on the patch, for example, "API_lyd_check_mandatory_tree.patch"

Reply:
Since we are not expecting many changes for libyang library, we have kept a single patch file for libyang changes related to management framework. Please suggest if current filename should be changed to any other generic name e.g. libyang_extension.patch or libyang_ext.patch.

Reply:
ext is not good,  libyang_mgmt_framework.patch is fine, but if we had to add in this file in the future then we need to break it down into multiple patches with description. 
That will be inline with other sonic repos, how we maintain patches. That will also help if some patches are accepted by upstream and we have to remove them in the future.